setup_cuda_deps.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+setup(
+    name='chainer-cuda-deps',
+    version='1.0.0',
+    description='A flexible framework of neural networks',
```

Comment:
It should be changed. How about 'Install dependent packages to use Chainer on CUDA'?